### DISCUSSION TITLE
Cannot connect to roomserver

### DISCUSSION THREAD

**Discussion by Yota73** [edumeet/edumeet-client#177] [Q&A]
Hello, I try edumeet compiling the single parts and rebuild in single docker images (and then full stack build with docker-compose) but I can't start in local development to try some changes.
I'm using a Windows 11 WSL2 ubuntu with nodejs 20.4. I've cloned the edumeet-client, edumeet-room-server and the edumemt-media-node and I yarn install all.
Then, after I create the room-server and the client config files I start the media-node, the room-server and the client (in this order)
In the vscode I see that the port 4443 and the port 3000 are forwarded to the localhost address and I can see the edumeet interface with the room name and the media selector.
But when I try to connect I got a websocket error.
I see that the link with the error is
wss://localhost:8443/socket.io/?peerId=......
so I try to put this link in postman and it works.
I also try to add the 8443 port in vscode redirect always to localhost (also try to use netsh to proxy other port to the 8443).
I use the test certificate provided with the code and I also try to remove the certificate but the connection in the link is a secure websocket so I don't think is the right way.

Any suggestion ?
Thanks

**Comment by havfo** [comment]
Go to `https://localhost:8443` in your browser to accept the certificate of the room-server. Then you should be able to go back to the client again and join.

**Reply by Yota73** [reply]
Thanks a lot. You save my day! 
(and  I'm feel a little stupid)
